shared/teams/team/settings-tab/index.tsx

```diff
@@ -270,6 +266,7 @@ export class Settings extends React.Component<Props, State> {
   }
 
   setBoolSettings = (key: any) => (newSetting: boolean) => {
+    // @ts-ignore not sure how to type this
```

Comment:
Maybe you could split `State` into the bool properties and the rest, and type `key: keyof BoolState` here. No need to do it now though.